core/tests/revert-test/tests/revert-and-restart.test.ts

```diff
@@ -125,19 +135,22 @@ describe('Block reverting test', function () {
         // Create test wallets
         tester = await Tester.init(ethClientWeb3Url, apiWeb3JsonRpcHttpUrl, baseTokenAddress);
         alice = tester.emptyWallet();
-        logs = fs.createWriteStream('revert.log', { flags: 'a' });
+        logs = fs.createWriteStream(`revert_${fileConfig.chain}.log`, { flags: 'a' });
     });
 
     step('run server and execute some transactions', async () => {
-        // Make sure server isn't running.
-        await killServerAndWaitForShutdown(tester).catch(ignoreError);
+        if (autoKill) {
+            // Make sure server isn't running.
+            await killServerAndWaitForShutdown(tester, serverProcess!).catch(ignoreError);
```

Comment:
`serverProcess!` could fail in runtime, because it's undefined here

Reply:
Addressed in https://github.com/matter-labs/zksync-era/pull/2594/commits/7879e2ea389135affb77c8c35585533c4bce2bea